src/ApiCompat/Rules/Compat/CannotSealType.cs

```diff
@@ -18,7 +18,7 @@ public override DifferenceType Diff(IDifferences differences, ITypeDefinition im
             if (impl.IsEffectivelySealed() && !contract.IsEffectivelySealed())
             {
                 differences.AddIncompatibleDifference(this,
-                    "Type '{0}' is sealed in the implementation but not sealed in the contract.", impl.FullName());
+                    $"Type '{impl.FullName()}' is sealed in the {Right} but not sealed in the {Left}.");
```

Comment:
I suspect I will have to keep looking whether Left/Right refers to contract/Implementation while working on the rules code.